src/components/dashboard/FeedItems/FeedModalItem.js

```diff
@@ -24,9 +25,9 @@ import SendModalItemWithError from './SendModalItemWithError'
  */
 const FeedModalItem = (props: FeedEventProps) => {
   const { item, onPress, styles, theme, navigation } = props
-  const buttonPress = () => {
+  const buttonPress = useOnPress(() => {
```

Comment:
```javascript
const buttonPress = useOnPress(() => onPress(item.id), [item, onPress])
```

here we're wrapping **some function using** onPress, not **just** onPress. 
useOnPress adds the _exactly function we're wrapping_ to the deps. If any prop or function is used _inside_, it _should_ be added as the dep